cfpb/cfgov-refresh
# Mega menu reorganization for IA release on 8/3

HOLD UNTIL 8/3
As part of the Information Architecture project, we're making some design and content changes to the mega menu (scheduled for release on 8/3), including the addition of footer links and headers for groups of menu items. 

## Changes

- Changes links in Consumer Tools and Practioner Resources sections.
- Updates mega menu data structure to add options for nav item group headers and footers.

## Testing

1. Check out branch and run gulp
2. Verify that mega menu works as expected

## Screenshots
<img width="1182" alt="screen shot 2017-07-19 at 11 51 15 am" src="https://user-images.githubusercontent.com/778171/28376700-c9952c06-6c78-11e7-856b-933a3447c59a.png">
<img width="1051" alt="screen shot 2017-07-19 at 11 51 28 am" src="https://user-images.githubusercontent.com/778171/28376709-cbd81dfc-6c78-11e7-89f4-444d99a19eb5.png">


## Notes

- I couldn't see a straightforward way to add the new data to the tuples in `_vars-mega-menu.html`, so I made some changes to the data structure (more in GHE 993) 

## Checklist
